app/javascript/.storybook/main.js

```diff
@@ -36,6 +37,13 @@ module.exports = {
       },
     };
 
+    if (configType === 'PRODUCTION') {
```

Comment:
Although Travis builds are the test environment, when you build the static storybook site, it’s environment internally gets set to `’PRODUCTION’`